### PR TITLE
Add link to the Zulip channel

### PR DESCRIPTION
**Pull request by triceo**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -5,3 +5,5 @@ gem 'rake', '>= 10.4.2'
 gem 'awestruct', '0.6.0'
 gem 'asciidoctor', '1.5.8' # AsciiDoc
 gem 'json', '2.1.0'
+gem 'pandoc-ruby'
+gem 'sassc'
```
Comment (yurloc) [inline Gemfile:9]: What are these new gems for? They don't seem to be related to the change.

Comment (triceo) [reply Gemfile:9]: Without these,  `rake build` would always result in a dependency error.

Comment (ge0ffrey) [reply Gemfile:9]: on your system? For me, "rake clean build" just works without it. Are you using the readme fedora approach of the "rvm" approach?

Comment (ge0ffrey) [reply Gemfile:9]: also, if you don't specify a verion number, does it use the "latest"? That's means our build could break from one day to the next. (and it's ruby, so it will)

**Comment by triceo** [comment]
Folks, the Gem changes are necessary, otherwise `rake clean build` results in an error.
I have followed the instructions precisely, and I consistently get this problem. (May be because my Fedora has Ruby 2.7, not 2.4?)

Anyway, we have two options here:
- Either we merge it and I can build the website on my machine,
- or we don't and you deal with the website yourselves, any way you please.

Your choice.

**Comment by ge0ffrey** [comment]
@triceo With versions numbers now, I am ok with it - merging.
And for the record: we're all responsible for the website!
